### PR TITLE
sparse assember new recipe

### PR DESCRIPTION
**Pull request by mictadlo**
* [x] I have read the guidelines above.
* [x] This PR adds a new recipe.
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/sparseassembler/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,34 @@
+{% set name = "SparseAssembler" %}
+{% set version = "20160205" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+build:
+  skip: True  # [osx]
```
Comment (bgruening) [inline recipes/sparseassembler/meta.yaml]: I'm not sure why you skip OSX if it seems to work.

File: recipes/sparseassembler/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  number: 0
+  script: "mkdir -p $PREFIX/bin; chmod a+x *; mv * $PREFIX/bin"
+
+source:
+  fn: SparseAssembler
+  url: https://raw.githubusercontent.com/yechengxi/SparseAssembler/master/compiled/SparseAssembler
```
Comment (bgruening) [inline recipes/sparseassembler/meta.yaml]: Can you compile this tool. precomiled things are not recommended.

File: recipes/sparseassembler/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+  md5: c9ba3f2dbbd9146767c3bdad02118a85
+
+requirements:
+  build:
+  - gcc [not osx]
```
Comment (bgruening) [inline recipes/sparseassembler/meta.yaml]: If you use precompiled binaries ,why are you using gcc?

**Comment by daler** [comment]
Looks like you're using a git repo, in which case you should be using `git_url` instead of `url` (http://conda.pydata.org/docs/building/meta-yaml.html#source-from-git).

Much better to use a release, but I don't see any source release in the repo. You might ask the author for a release tag. But please at least use a commit hash rather than master, since master branch can change at any time making the recipe non-reproducible.

**Comment by mictadlo** [comment]
@daler @bgruening I changed it, but now I get `SparseAssembler: command not found`. What could I miss?

File: recipes/sparseassembler/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,33 @@
+{% set name = "SparseAssembler" %}
+{% set version = "20160205" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+build:
+  number: 0
+  script: "mkdir -p $PREFIX/bin; g++ -O3 -o SparseAssebmler *.cpp; mv SparseAssebmler $PREFIX/bin"
```
Comment (daler) [inline recipes/sparseassembler/meta.yaml]: Looks like typos here, s/ebm/emb/g. That will cause the error you're seeing on travis.

**Comment by daler** [comment]
Getting there! Hopefully those typo fixes will do the trick.

**Comment by mictadlo** [comment]
Thank you @bgruening  and @daler for your help

**Pull request merged**